packages/cli/src/build/index.js

```diff
@@ -88,9 +88,10 @@ async function build({flags}) {
   // Create /
   if (typeof cover === "string") {
     const cover = getModule(COVER_PATH);
+    const context = getCoverContext(config);
     const result = typeof cover.render === "function"
-      ? cover.render(cover)
-      : getModule(RENDER_PATH)(cover);
+      ? cover.render(cover, context)
```

Comment:
Would be great if we could await those, too.

Reply:
I thinks binding the relevant function to a `render` is a good idea in all cases, that should help with readability when applying the `Promise.resolve` guard for async execution

Reply:
Changed in new commit.